modules/course_management/cog.py

```diff
@@ -4,15 +4,24 @@
 from discord_slash.context import SlashContext
 from discord_slash.model import SlashCommandOptionType
 from discord_slash.utils.manage_commands import create_option
+from modules.course_management import util
+from modules.course_management.util import sort_courses
 from utils.embedder import embed_success
 from discord.ext.commands import has_permissions
 from discord.ext import commands
+from utils.utils import get_discord_obj
 from . import course_adder
 from . import course_deleter
+from . import course_activator
 import config
+from discord.ext import tasks
 
 
 class CourseManagerCog(commands.Cog):
+	@commands.Cog.listener()
+	async def on_ready(self):
+		self.sort_courses_categories.start()
```

Comment:
[on_ready](https://discordpy.readthedocs.io/en/stable/api.html?highlight=on_ready#discord.on_ready) can be triggered multiple times. If it's an issue for this to be called multiple times, there should probably be a flag to check if it's been started.